modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/exportimport/portlet/preferences/processor/DLExportImportPortletPreferencesProcessor.java

```diff
@@ -310,6 +321,23 @@ public PortletPreferences processImportPortletPreferences(
 				try {
 					portletPreferences.setValue(
 						"rootFolderId", String.valueOf(rootFolderId));
+
+					Folder folder = _dlAppLocalService.getFolder(rootFolderId);
+
+					portletPreferences.setValue(
+						"selectedRepositoryId",
+						String.valueOf(folder.getRepositoryId()));
+				}
+				catch (PortalException portalException) {
+					StringBundler sb = new StringBundler(2);
+
+					sb.append("Invalid root folder ID ");
+					sb.append(rootFolderId);
+
+					_log.error(sb.toString());
+
+					throw new PortletDataException(
+						sb.toString(), portalException);
```

Comment:
Did SF complain about the usage of string bundler? Otherwise, unless there are more than 3 concatenations, you can use the regular `+` to concatenate the error message.